client/src/components/InlinePanel/index.js

```diff
@@ -226,6 +256,7 @@ export class InlinePanel extends ExpandingFormset {
       /__prefix__/g,
       formIndex,
     );
+    this.addTabIndex(newChildPrefix);
```

Comment:
Let's move this to the last thing in this method. After all the run Callbacks have fired. 

This way any Callbacks that add DOM or listeners can work before the focus code runs. 

Also, rename to `initialFocus`